[KB-288] Experiment design for initial experiment (#200)
chhoumann/thesis-chemcam: report_thesis/src/sections/experiment_design/experimental_setup.tex

\subsection{Experimental Setup}\label{sec:experimental_setup}
Experiments were conducted on a machine equipped with an Intel Xeon Gold 6242 CPU, featuring 16 cores and 32 threads.
The CPU has a base clock speed of 2.80 GHz and a maximum turbo frequency of 3.90 GHz.
The system has 64 GB of RAM and runs on Ubuntu 22.04.2 LTS.
Models were implemented using Python 3.10.11.
The primary libraries used were Scikit-learn 1.4.2, XGBoost 2.0.3, Torch 2.2.2, NumPy 1.26.4, Pandas 2.2.1, Keras 3.2.1 and Optuna 3.6.1.

> Additionally, all experiments were run using the hyperparameter optimization tool described in Section~\ref{subsec:hyperparameter_tuning_tool}. % TODO: Add correct ref once other PR is in

Author: ```suggestion
Additionally, all experiments were run using the hyperparameter optimization tool described in Section~\ref{sec:optimization_framework}.
```
